worlds/dkc3/Client.py

```diff
@@ -33,14 +33,14 @@ async def deathlink_kill_player(self, ctx):
     async def validate_rom(self, ctx):
         from SNIClient import snes_buffered_write, snes_flush_writes, snes_read
 
-        game_hash = await snes_read(ctx, DKC3_ROMHASH_START, ROMHASH_SIZE)
-        if game_hash is None or game_hash == bytes([0] * ROMHASH_SIZE) or game_hash[:2] != b"D3":
+        rom_name = await snes_read(ctx, DKC3_ROMHASH_START, ROMHASH_SIZE)
```

Comment:
This particular rename should be reverted for DKC3.

Also, per the conclusion of our old discussion, DKC3_ROMHASH_START should be moved away from 0x7fc0 to avoid confusing softwares that operate on based on ROM layout, such as emulators and SNI.

Reply:
It's the same variable doing the same functionality as in all of the other Clients, so it will have the same name.

As for confusing softwares, neither SNI nor any emulators I've encountered have had any issues with it. Perhaps someday it will get moved, but that's not in-scope here.